docs/source/deploy/ledger-topologies.rst

```diff
@@ -69,7 +69,7 @@ Scaling
 
 The main functionalities of a system providing the Ledger API are:
 
-1. serving the API itself (handling the gRPC connections, authenticating users, etc),
+1. serving the API itself: handling the gRPC connections and authorizing users,
```

Comment:
Not quite sure if this is exhaustive but I guess we can add more later if we find something concrete.